src/config.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
```

Comment:
`constants` would be a more appropriate name, given that these are not meant to be changed